### DISCUSSION TITLE
Bulk creation of posts

### DISCUSSION THREAD

**Discussion by agraash** [austintoddj/canvas#1162] [General]
Hi

Is it possible to create posts in bulk using the backend itself. 

As a part of my project requirements, I need to create posts in bulk and it is rather cumbersome to use the editor to set the posts up, one at a time.

Thank you for your time

Ash

**Comment by austintoddj** [comment]
Are you importing posts from somewhere else? I'm curious what sort of use case would lead you to wanting to bulk create posts.
